components/modal/checkpointsModal.tsx

```diff
@@ -101,19 +103,20 @@ export default function CheckpointsModal({ closeModal, isOpen }: CheckpointsModa
       title={'Checkpoints'}
     >
       {isPro(user) ?
-        <div className='flex flex-col gap-4 max-w-full'>
+        <div className='flex flex-col gap-4 '>
           <span className='text-center'>
             Keyboard shortcuts for slots 0-9:<br />
             Save: Shift + 0-9<br />
             Load: 0-9
           </span>
-          <div className='flex flex-col gap-1 w-80 max-w-full justify-center'>
+          <div className='flex flex-col gap-1 justify-center md:grid md:grid-cols-2 md:gap-4'>
             {checkpoints?.map((checkpoint, i) => (
               <CheckpointImage
                 checkpoint={checkpoint}
                 closeModal={closeModal}
                 key={'checkpoint-' + i}
                 slot={i}
+                title={i === USER_BEST_MOVE_CHECKPOINT_SLOT ? 'Your Best' : undefined}
```

Comment:
I think we should always show `USER_BEST_MOVE_CHECKPOINT_SLOT` as the first item. Can use the 10th index in the db to avoid any datafixing but 0th index for the UI is better imo

Reply:
Also, users should not be able to overwrite this checkpoint. It should purely be handled by the backend

Reply:
Ah yes this is better approach...